### DISCUSSION TITLE
Duty Cycle Dopplung seit Update vom 13. November

### DISCUSSION THREAD

**Discussion by martinflux** [SukramJ/aiohomematic#1871] [Q&A] [answered]
Hallo zusammen,

vorab vielen Dank für diese großartige Integration und Eure fortlaufenden Mühen, sie aktuell zu halten!

Mein Thema: Seit dem 13.11. stelle ich einen deutlichen Anstieg meines Duty Cycle fest (grob eine Verdopplung). Meine Fehlersuche ist bisher erfolglos und auch das Deaktivieren von HA Automationen hat nicht zu einer Linderung geführt. Änderungen innerhalb Raspberymatic hat es in diesem Zeitraum nicht gegeben.

Mein Verdacht ist, dass sich die Verdopplung mit dem hahomematic Update (~ Release 2024.11.1?) eingeschlichen hat.

Daher die Frage: Besteht die Möglichkeit, dass dies mit dem Release zusammenhängt? Beobachten ggf. andere Nutzer hier ebenso einen Anstieg des DC?

Danke im Voraus und Grüße,
Martin

![image](https://github.com/user-attachments/assets/9b54c5cb-a343-4953-855c-f66b873a5cc8)

### What version of HomematicIP (local) has the issue?

1.71.0

### What was the last working version of HomematicIP (local)?

1.6x.x

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic Standalone

**Comment by SukramJ** [comment, marked as answer]
Immer wieder kommen solche Anfragen, und am Ende stellt sich heraus, das ein Gerät verrückt spielt.

Egal.

Die Integration fragt beim Start Daten von der CCU ab, wobei der der Grossteil aus der Rega, also ohne Gerätzugriff abgefragt wird.
Danach sendet die CCU Events mit den Statusänderungen an HA. 

Eine dauerhafte Abfrage von Gerätedaten findet nicht statt.


> Mein Verdacht ist, dass sich die Verdopplung mit dem hahomematic Update (~ Release 2024.11.1?) eingeschlichen hat.

Die Releases der Integration findest du [hier](https://github.com/danielperna84/custom_homematic/releases). Dann wirst Du auch feststellen, das es am 13. November kein Release der Integration gab.

**Reply by martinflux** [reply]
Danke für Deine schnelle Antwort und nichts für Ungut, Sukram! Weil mein Setup unverändert ist, lag der Verdacht nahe....

Bin dann mal gespannt, ob ich dem Thema noch auf die Schliche komme. Ggf. liegt es auch daran, dass die Thermostate aufgrund der Witterung gerade viel zu tun bekommen und mein HM-ES-TX-WM ordentlich funkt.

Viele Grüße!

**Comment by stuckis6er** [comment]
Hallo, kann es sein das evtl. Geräteupdates laufen? Sieht so aus wie bei mir wenn Updates laufen.
![Screenshot_20241122_130429_Home Assistant.jpg](https://github.com/user-attachments/assets/c2edc5fb-3bf2-4e1b-a260-00901e187715)
